docs/docs-content/getting-started/introduction.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: "Introduction to Palette"
+title: "Introduction to Palette"
+description: "Learn what Spectro Cloud's Palette platform is."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["getting-started"]
+---
+
+Palette is a complete and integrated platform that enables organizations to effectively manage the entire lifecycle of
+any combination of new or existing, simple or complex, small or large Kubernetes environments, whether in a data center
+or the cloud.
+
+With a unique approach to managing multiple clusters, Palette gives IT teams complete control, visibility, and
+production-scale efficiencies to provide developers with highly curated Kubernetes stacks and tools based on their
+specific needs, with granular governance and enterprise-grade security.
+
+Palette VerteX edition is also available to meet the stringent requirements of regulated industries such as government
+and public sector organizations. Palette VerteX integrates Spectro Cloud’s Federal Information Processing Standards
+(FIPS) 140-2 cryptographic modules. To learn more about FIPS-enabled Palette, check out
+[Palette VerteX](../vertex/vertex.md).
+
+![Palette product high level overview eager-load](/getting-started/getting-started_introduction_product-overview.png)
+
+## What Makes Palette Different?
+
```

Comment:
Let's add something between these two headings. Right now, it's a bit odd that we have back-to-back headings. (_It's been like this on our main intro site for a long time_)